app/components/chart/modalsubnav.js

```diff
@@ -109,20 +114,20 @@ var ModalSubNav = React.createClass({
     /* jshint ignore:start */
     return (
       <div>
+        <div className="domainContainer">{domainLinks}</div>
         <div className="visibleDays">{visibleDaysText}</div>
-        <div>{domainLinks}</div>
       </div>
       );
     /* jshint ignore:end */
   },
   renderDomainLink: function(domain) {
     var domainLinkClass = cx({
-      'active': domain === this.props.activeDomain,
-      'modalDomain': true
+      'btn btn-chart-control' : true,
+      'active': domain === this.props.activeDomain
     });
     /* jshint ignore:start */
     return (
-      <a href="" className={domainLinkClass} key={domain} onClick={this.props.domainClickHandlers[domain]}>{domain}</a>
```

Comment:
Can you put the `key={domain}` back? Without it you get a React warning in the console (you may need React dev tools installed to see these): "Each child in an array should have a unique "key" prop."